lib/getData.py

```diff
@@ -0,0 +1,103 @@
+import os
+import fire
+import shutil
+
+import numpy as np
+import astroquery.mast as mast
+import astropy.io.fits as pyfits
+
+def getData (pi, target, pid, sciFil, imgFil, fileSearch='ima.fits', local='.', sortPath='.'):
```

Comment:
Are all of these inputs strictly necessary?  I feel like some/all should be optional, with default being `None`.

Reply:
`pi`, `target`, and `pid` are search parameters and strictly necessary
`sciFil` and `imgFil` are dataset specific and while they could have defaults you'd have to tell me that >50% of the time this is going to be used on G141 and F126N.
Everything else is optional.

Reply:
What I was trying to say was that the `query_criteria` function doesn't explicitly require these three parameters.  There's a long list of acceptable inputs, so the best way to handle this might be to use `**kwargs` in `getData` that get passed directly to `query_criteria`.  Does that make sense?  In the description below, we could then list some of the most helpful keyword arguments (e.g., proposal_pi, proposal_id, objID, obs_id, target_name, filters, instrument_name, etc).